src/server/config/schema.js

```diff
@@ -232,6 +232,11 @@ export default async () => Joi.object({
       then: Joi.string().default('https://staging-dot-catalogue-dot-elastic-layer.appspot.com/v2/manifest'),
       otherwise: Joi.string().default('https://catalogue.maps.elastic.co/v2/manifest')
     }),
+    emsLandingPageUrl: Joi.when('$dev', {
+      is: true,
+      then: Joi.string().default('https://maps-staging.elastic.co/v2'),
+      otherwise: Joi.string().default('https://maps.elastic.co/v2')
```

Comment:
Discussed with @thomasneirynck previously, the current manifest used to populate the dropdown reports slightly different layers than what's available at the current EMS endpoints, for example 'USA Counties' is listed as available via the manifest, but the dynamically generated link with that layer name argument provided doesn't actually go to the resource. This is a known mismatch originally identified by @nickpeihl, and is something we can tackle either as a part of this issue or it could be a separate issue.

Reply:
I'm confident this will be a separate issue and doesn't necessarily have to hold up merging for now so long as the external services are updated and we update the manifest used for the dropdown and the generated link. I ran the code locally with no apparent issues. LGTM!

Reply:
This is a separate issue, mainly because I misunderstood how the EMS landing page deployment works. maps-staging.elastic.co/v2 is showing the production manifest rather than the staging manifest.

Reply:
we decided to just have Kibana use production-EMS all the time, even in dev/master. https://github.com/elastic/kibana/pull/21237 addresses that (and should be merged before this one).